components/post_view/post_list.jsx

```diff
@@ -116,10 +130,41 @@ export default class PostList extends React.PureComponent {
         window.addEventListener('resize', this.handleWindowResize);
     }
 
-    componentDidUpdate(prevProps) {
+    getSnapshotBeforeUpdate(prevProps, prevState) {
```

Comment:
I was unable to add a test case for this @saturninoabril any suggestions?

Reply:
One approach is to mock return value of `postlistRef` and watch for the snapshot at `componentDidUpdate` whenever it's called.  Maybe something like, 

```javascript
test('should return previous scroll position from getSnapshotBeforeUpdate', () => {
    const wrapper = shallow(<PostList {...baseProps}/>);
    const instance = wrapper.instance();
    instance.componentDidUpdate = jest.fn();

    instance.postlistRef = {current: {scrollTop: 10, scrollHeight: 100}};
    wrapper.setState({atEnd: true});
    expect(instance.componentDidUpdate).toHaveBeenCalledTimes(1);
    expect(instance.componentDidUpdate.mock.calls[0][2]).toEqual({previousScrollTop: 10, previousScrollHeight: 100});

    instance.postlistRef = {current: {scrollTop: 30, scrollHeight: 200}};
    wrapper.setState({atEnd: false});
    expect(instance.componentDidUpdate).toHaveBeenCalledTimes(2);
    expect(instance.componentDidUpdate.mock.calls[1][2]).toEqual({previousScrollTop: 30, previousScrollHeight: 200});

    // may add test combination of state changes necessary in computing getSnapshotBeforeUpdate
});
```

Reply:
Added test cases but as discussed offline downgrading enzyme because of https://github.com/airbnb/enzyme/pull/2027